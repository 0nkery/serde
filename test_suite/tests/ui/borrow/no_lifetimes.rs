#[macro_use]
extern crate serde_derive;

#[derive(Deserialize)]
struct Test {
    #[serde(borrow)]
    s: String,
}

fn main() {}
