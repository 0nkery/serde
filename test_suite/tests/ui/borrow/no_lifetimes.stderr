error: field `s` has no lifetimes to borrow
 --> $DIR/no_lifetimes.rs:4:10
  |
4 | #[derive(Deserialize)]
  |          ^^^^^^^^^^^

error: aborting due to previous error

